src/openloco/config.cpp

```diff
@@ -54,27 +67,30 @@ namespace openloco::config
 
     new_config& read_new_config()
     {
-        auto configPath = environment::get_path(environment::path_id::openloco_cfg);
+        auto configPath = environment::get_path(environment::path_id::openloco_yml);
+
 #ifdef _OPENLOCO_USE_BOOST_FS_
-        std::ifstream stream(configPath.string());
+        YAML::Node config = YAML::LoadFile(configPath.string());
 #else
-        std::ifstream stream(configPath);
+        // WARNING: YAML::LoadFile only supports ANSI 260 paths
```

Comment:
This comment is wrong, it should be removed.

Reply:
It may well be true for Windows, though. I'd assumed @IntelOrca added it because he got a warning when compiling.

Reply:
I didn't get a warning. I checked the source code for yaml-cpp and saw that they are using the ANSI overload for ifstream. I don't know why @janisozaur is saying it's wrong unless he is just being pedantic about it not saying Windows only.

Reply:
Because the comment was wrong. I created a file with 270-chars long path and it was read correctly.

Reply:
Then the comment is still half right as it still limited to ANSI, it does not allow Unicode.

Reply:
MAX_PATH is still 260 characters [on Windows](https://docs.microsoft.com/en-us/windows/desktop/fileio/naming-a-file#maximum-path-length-limitation) though, right? Since the comment has been amended to say Windows already, I think no further change is required.